main.go

```diff
@@ -904,3 +902,28 @@ func startGcsMigrator() {
 	}
 	gcsMigratorStartedAt = time.Now()
 }
+
+func getKubeClientConfig() *rest.Config {
```

Comment:
I think you can replace this logic by using getConfigOrDie from controller config runtime package 
"sigs.k8s.io/controller-runtime/pkg/client/config" . This will handle in/out cluster.

Reply:
Thanks for letting me know @afarid 

My original idea was to run `estafette-ci-api` and `estafett-ci-web` independently outside the cluster and it might be easier to locally develop and see how it works.

Just saw [the doc](https://estafette.io/development/) that develop using Helm way.